openml/tasks/functions.py

```diff
@@ -450,3 +450,60 @@ def _create_task_from_xml(xml):
         raise NotImplementedError('Task type %s not supported.' %
                                   common_kwargs['task_type'])
     return cls(**common_kwargs)
+
+
+def create_task(
+        task_type_id: int,
+        dataset_id: int,
+        target_name: str,
+        estimation_procedure_id: int,
+        evaluation_measure: str = None,
+        **kwargs
+):
```

Comment:
Initially, I had an `OpenMLTask` return type, since it is the base class. However, the class can return different task types based on the task type id, so `Union[task_type_1, ..., task_type_5]`  is a viable option, but too specific. I was not sure on what notation to use.

Reply:
Why would that be too specific? I think the union would be the best option here.

Reply:
Then specific would be the wrong word used by me in this situation. What I meant, was that given that there are 4 possible classes, the return would be: 
`Union[class1, Union[class2, Union[class3, class4]]]`
In the future adding more classes would add more Unions. So I am not sure if this appears confusing compared to having the base class, which in turn does not showcase all the task types.
I think the Union is the best option too and since you agree, I am going with that one.

Reply:
I think that's the best decision at the moment. We can still update this if when there are more classes coming up.

Regarding the specific annotation, doesn't `Union[class1, class2, class3, class4]` work as well?

Reply:
> I think that's the best decision at the moment. We can still update this if when there are more classes coming up.
> 
> Regarding the specific annotation, doesn't `Union[class1, class2, class3, class4]` work as well?

more than right, I read the documentation for the redundant arguments by mistake and I did not pay attention. I thought there was a limit. Good catch.